lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -1237,6 +1473,17 @@ void HdVP2Material::_UpdateShaderInstance(const HdMaterialNetwork& mat)
                 MFloatMatrix matrix;
                 value.UncheckedGet<GfMatrix4f>().Get(matrix.matrix);
                 status = _surfaceShader->setParameter(paramName, matrix);
+            } else if (value.IsHolding<std::string>()) {
```

Comment:
I think I would prefer that the else if be inside the WANT_MATERIALX_BUILD macro. It is just an empty clause now which looks strange to me.

Reply:
+1